src/mlpack/core/data/scaler_methods/maxabsscaler.hpp

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file maxabsscaler.hpp
+ * @author Jeffin Sam
+ *
+ * MaxAbsScaler class to scale features.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_MAX_ABS_SCALE_HPP
+#define MLPACK_CORE_DATA_MAX_ABS_SCALE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
+/**
+ * A simple MaxAbs Scaler class
+ */
+template <typename T>
+class MaxAbsScaler
+{
+ public:
+  /**
+  * Default constructor
+  *
+  */
+  MaxAbsScaler(){}
+  /**
+  * Default Destructor 
+  */
+  ~MaxAbsScaler(){}
+
+  /**
+  * Function to scale Features.
+  *
+  * @param input Datset to scale features.
+  */
+  void Transform(arma::Mat<T>& input)
```

Comment:
Do you think we should provide an option to save the result into another matrix?

Reply:
I don't mind it, Changed it accordingly